src/facade/dragonfly_connection.h

```diff
@@ -80,8 +80,8 @@ class Connection : public util::Connection {
   struct MonitorMessage : public std::string {};
 
   struct AclUpdateMessage {
-    std::string_view username;
-    uint64_t categories{0};
+    std::vector<std::string> username;
```

Comment:
This was also a bug, `std::string_view` could have been deleted if the queue item was not consumed and the string_view associated with the name got deallocated (e.g, user deletion)

Reply:
I remember I asked in the previous PR 😆

Reply:
Yes and I replied wrongly because I did not think about this case. It's fine, we caught it and I now know better. 

P.s. you don't forget do you ? :eyes:

Reply:
No 🤣